### PR TITLE
using  OemrUI class  in Fees > Fee Sheet, Billing, Payment, Checkout, Batch Payments, Posting

### PR DESCRIPTION
**Pull request by zbig01**
Converted all user facing files in Fees to use the OemrUI class

### CODE REVIEW

File: interface/billing/new_payment.php [hunk 1]
```diff
@@ -408,21 +409,32 @@ function FillUnappliedAmount()
     //become the user-specific default for that page. collectAndOrganizeExpandSetting() contains a single array as an
     //argument, containing one or more elements, the name of the current file is the first element, if there are linked
     // files they should be listed thereafter, please add _xpd suffix to the file name
-    $arr_files_php = array("new_payment_xpd", "search_payments_xpd", "era_payments_xpd");
-    $current_state = collectAndOrganizeExpandSetting($arr_files_php);
-    require_once("$srcdir/expand_contract_inc.php");
+    // $arr_files_php = array("new_payment_xpd", "search_payments_xpd", "era_payments_xpd");
+    // $current_state = collectAndOrganizeExpandSetting($arr_files_php);
+    // require_once("$srcdir/expand_contract_inc.php");
```
Comment (bradymiller) [inline interface/billing/new_payment.php]: better to remove than uncomment

File: interface/patient_file/front_payment.php [hunk 2]
```diff
@@ -946,17 +947,33 @@ function make_insurance() {
 }
 </style>
 <title><?php echo xlt('Record Payment'); ?></title>
+<?php $NameNew = $patdata['fname'] . " " . $patdata['lname'] . " " . $patdata['mname']; ?>
```
Comment (bradymiller) [inline interface/patient_file/front_payment.php]: Is the above $NameNew variable being used?

Comment (bradymiller) [reply interface/patient_file/front_payment.php]: disregard above comment, see it being used below

**Comment by bradymiller** [comment]
hi @zbig01 , Only commented on a minor issue. If this is testing well for you, just let me know, and then I'll bring this into the codebase. thanks! -brady

**Comment by bradymiller** [comment]
oops, closed this by mistake :)

File: interface/patient_file/front_payment.php (hunk 2 above)
Comment (zbig01) [reply interface/patient_file/front_payment.php]: @bradymiller I am not sure what it is doing here as this is redefined on line 166.  It was there in the original script in the <h4></h4> tag and therefore  I left it there :)

**Comment by zbig01** [comment]
@bradymiller 

Working well, no change to the logic :)

**Comment by bradymiller** [comment]
hi @zbig01 , bringing into codebase now. thanks! -brady

**Pull request merged**